var/spack/repos/builtin/packages/lfortran/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Lfortran(Package):
+    """LFortran"""
```

Comment:
Could be a little more precise
```suggestion
    """Modern interactive LLVM-based Fortran compiler"""
```